lib/fileUpload.coffee

```diff
@@ -39,7 +41,8 @@ if UploadFS?
 			onFinishUpload: ->
 				console.log arguments
 			onRead: (fileId, file, req, res) ->
-				res.setHeader 'content-disposition', 'download'
+				unless file?.type?.match(/^image\/.*/)
```

Comment:
As we have a preview inside the application I think we can just remove the condition and force all files to be downloaded when accessed direct.